.gitignore

```diff
@@ -1,20 +1,3 @@
-dist
-cabal-dev
-*.o
-*.hi
-*.chi
-*.chs.h
-*.dyn_o
-*.dyn_hi
-.virtualenv
-.hpc
-.hsenv
-.cabal-sandbox/
-cabal.sandbox.config
-*.prof
-*.aux
-*.hp
-.DS_Store
-.stack-work/
-mustache.cabal
```

Comment:
These are either outdated and not needed anymore, or they are a better fit for a [global git ignore file](https://docs.github.com/en/get-started/git-basics/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).